### ISSUE TITLE
Add nested name spaces support to python bindings generator

### ISSUE THREAD

**Issue by opencv-pushbot** [Itseez/opencv#4768]
Transferred from http://code.opencv.org/issues/3745
```
|| Alexander Smorkalov on 2014-06-10 06:13
|| Priority: Normal
|| Affected: None
|| Category: python bindings
|| Tracker: Feature
|| Difficulty: 
|| PR: 
|| Platform: None / None
```

Add nested name spaces support to python bindings generator
-----------
```

```

History
-------

**Comment by Dikay900** [comment]
nested namesspaces are available now as far as i know example
cv2.xfeatures2d. etc.
so this can be closed

**Comment by mshabunin** [comment]
Implemented in #3878
